app/src/main/java/com/samco/trackandgraph/util/UtilFuncs.kt

```diff
@@ -75,16 +77,48 @@ fun Context.getColorFromAttr(
     return typedValue.data
 }
 
+fun View.focusAndShowKeyboard() {
```

Comment:
Keyboard management is such a shit show on Android. These best functions for showing the keyboard change every year XD .. thanks for this though it looks superior.